README.md

```diff
@@ -8,8 +8,8 @@ Sometimes your rows might come in a nested format or it might have a representat
 import * as resolve from 'table-resolver';
 
 // Or you can cherry-pick
-import { index } from 'table-resolver';
-import { index as resolveIndex } from 'table-resolver';
+import { nested } from 'table-resolver';
+import { nested as resolveNested } from 'table-resolver';
 ```
 
 ## API
```

Comment:
Yes. I didn't find any instances of it being used anyway. I changed it so that `resolver.resolve` always adds `_index` to every resolved row. Although I now see that may be problematic; what if someone has a column legitimately named "index" and they use `byFunction`? Maybe we could make that optional?